addons/crm/models/crm_lead.py

```diff
@@ -184,7 +184,7 @@ def _compute_day_open(self):
         for lead in self.filtered(lambda l: l.date_open and l.create_date):
             date_create = fields.Datetime.from_string(lead.create_date)
```

Comment:
ok in general, but we could simplify by having just something like 
`date_create = fields.Datetime.from_string(lead.create_date).replace(microsecond=0)`
since the microseconds does not seems to be present in date_open

Reply:
This is what I wanted to do but I thought that date_open can have microsecond if set in xlxs on import. 
Just tried and since it's a read-only field, your fix will work better. Thanks!